charts/ctlog/values.yaml

```diff
@@ -0,0 +1,59 @@
+namespace:
+  create: false
+  name: ctlog-system
+server:
+  replicaCount: 1
+  image:
+    registry: ghcr.io
+    repository: vaikas/ct_server
+    pullPolicy: IfNotPresent
```

Comment:
I think this might cause hilarity on Kind installations, so wonder if we should just remove this?